client/src/main/java/io/pravega/client/netty/impl/ClientConnectionImpl.java

```diff
@@ -72,10 +73,10 @@ public void sendAsync(WireCommand cmd, CompletedCallback callback) {
                        }
                    });
         } catch (ConnectionFailedException cfe) {
-            log.debug("ConnectionFaileException observed when attempting to write WireCommand {} ", cmd);
+            log.debug("ConnectionFailedException observed when attempting to write WireCommand {} ", cmd);
             callback.complete(cfe);
         } catch (Exception e) {
-            log.warn("Exception while attempting to write WireCommand {} on netty channel {}", cmd, channel);
+            log.warn("Exception while attempting to write WireCommand {} on netty channel {}", cmd, channel, e);
```

Comment:
Should we output the whole stacktrace or just `e.toString()`?

Reply:
removed this as the exception is logged by the users of `io.pravega.client.netty.impl.ClientConnectionImpl#sendAsync()`